webapp/src/components/sidebar/playbooks_sidebar.tsx

```diff
@@ -47,9 +49,57 @@ const PlaybooksSidebar = (props: PlaybookSidebarProps) => {
                 }) : [],
             };
         });
+        addViewAllsToGroups(calculatedGroups);
         return calculatedGroups;
     };
 
+    const addViewAllsToGroups = (groups: SidebarGroup[]) => {
+        for (let i = 0; i < groups.length; i++) {
+            if (groups[i].id === 'runsCategory') {
+                groups[i].customLastItem = viewAllRuns();
+            } else if (groups[i].id === 'playbooksCategory') {
+                groups[i].customLastItem = viewAllPlaybooks();
+            }
+        }
+    };
+
+    const {formatMessage} = useIntl();
+    const viewAllMessage = formatMessage({defaultMessage: 'View all...'});
+
+    const viewAllRuns = (): ReactNode => {
```

Comment:
Nit: is this explicit typing needed?
```suggestion
    const viewAllRuns = () => {
```